pages/common/birdc.md

```diff
@@ -0,0 +1,37 @@
+# birdc
+
+> Bird remote control.
+> Command tool to retrieve information like routes from bird and perform configurations during runtime.
+> More information: <https://bird.network.cz/>.
+
+- Open the remote control console::
+
+`birdc`
+
+- Reload the configuration without restarting Bird:
+
+`birdc configure`
+
+- Show the current status of Bird:
+
+`birdc show status`
+
+- Show all active protocols:
+
+`birdc show protocols`
+
+- Show all details about a protocol:
+
+`birdc show protocols {{upstream1}} all`
```

Comment:
It would be better if you give an example value of `upstream1` in the description. I saw `all` is a valid value.

Reply:
How exactly do you mean? `upstream1` is just a placeholder for a BGP protocol configured in bird. `all` causes not only the status but all details of the protocol to be displayed.

Reply:
Sorry, I read it wrong. Never mind.